tools/deb/postinst.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Done. Not sure why actually - env bash is supposed to be more portable, I've seen systems where bash is in /usr/bin without a link in bin. But long ago.